### PR TITLE
Syncing for #1304

### PR DESCRIPTION
**Pull request by martyav**
Fixes #[issue_number]

## Reminders

- See the [README](../README.md) for more details on how to work with the Rancher docs.

- Verify if changes pertain to other versions of Rancher. If they do, finalize the edits on one version of the page, then apply the edits to the other versions.

- If the pull request is dependent on an upcoming release, make sure to target the release branch instead of `main`.

This syncs changes across versions for #1304

### CODE REVIEW

**Review by btat — CHANGES_REQUESTED** [review]

File: versioned_docs/version-2.8/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md [hunk 1]
```diff
@@ -21,65 +21,48 @@ To interact with Azure APIs, an AKS cluster requires an Azure Active Directory (
 Before creating the service principal, you need to obtain the following information from the [Microsoft Azure Portal](https://portal.azure.com):
 
 - Subscription ID
-- Client ID
+- Client ID (also known as app ID)
```
Comment (btat) [inline versioned_docs/version-2.8/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md:24]:
```suggestion
- Client ID (also known as app ID)
```

File: versioned_docs/version-2.7/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md [hunk 2]
```diff
@@ -21,65 +21,48 @@ To interact with Azure APIs, an AKS cluster requires an Azure Active Directory (
 Before creating the service principal, you need to obtain the following information from the [Microsoft Azure Portal](https://portal.azure.com):
 
 - Subscription ID
-- Client ID
+- Client ID (also known as app ID)
```
Comment (btat) [inline versioned_docs/version-2.7/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md:24]:
```suggestion
- Client ID (also known as app ID)
```

File: docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md [hunk 3]
```diff
@@ -21,65 +21,48 @@ To interact with Azure APIs, an AKS cluster requires an Azure Active Directory (
 Before creating the service principal, you need to obtain the following information from the [Microsoft Azure Portal](https://portal.azure.com):
 
 - Subscription ID
-- Client ID
+- Client ID (also known as app ID)
```
Comment (btat) [inline docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/aks.md:24]:
```suggestion
- Client ID (also known as app ID)
```

**Review by btat — COMMENTED** [review]
What's the reason for having a different order for this line?
<= 2.6: `An app ID (also called a client ID)`
>= 2.7: `Client ID (also known as app ID)`

**Comment by martyav** [comment]
> What's the reason for having a different order for this line? <= 2.6: `An app ID (also called a client ID)` >= 2.7: `Client ID (also known as app ID)`

That wasn't the intention, just pushed a commit to correct the error

**Review by btat — APPROVED** [review]

**Pull request merged**